localstack-core/localstack/utils/crypto.py

```diff
@@ -4,7 +4,15 @@
 import re
 import threading
 
+from asn1crypto import algos, cms, core
+from asn1crypto import x509 as asn1_x509
+from Crypto.Cipher import AES, PKCS1_OAEP
+from Crypto.Hash import SHA256
+from Crypto.PublicKey import RSA as CryptoRSA
+from Crypto.Random import get_random_bytes
+from Crypto.Util.Padding import pad as crypto_pad
```

Comment:
using `from cryptography.hazmat.primitives.asymmetric import padding` should suffice the purpose here as well.